release-testing/prepare.sh

```diff
@@ -87,19 +137,19 @@ x509_extensions = v3_ca
 basicConstraints = critical,CA:TRUE
 subjectKeyIdentifier   = hash
 authorityKeyIdentifier = keyid:always,issuer:always
-'
+' > ca.config
```

Comment:
This should be probably in both dockerignore (if applicable) and gitignore

Reply:
Probably same thing with ca.key/ca.cert

Reply:
These are run inside Kubernetes pod so both not applicable